samples/bluetooth/shell_bt_nus/src/main.c

```diff
@@ -61,7 +61,11 @@ static void disconnected(struct bt_conn *conn, u8_t reason)
 static void __attribute__((unused)) security_changed(struct bt_conn *conn,
 						     bt_security_t level)
 {
-	LOG_INF("Security level was raised to %d", level);
+	char addr[BT_ADDR_LE_STR_LEN];
+
+	bt_addr_le_to_str(bt_conn_get_dst(conn), addr, sizeof(addr));
+
+	LOG_INF("Security changed: %s level %u\n", addr, level);
```

Comment:
Yes @KAGA164, I would change this to `printk` to keep it aligned with other samples.
Remeber that `LOG` requires `log_strdup` to log strings anyway.

Reply:
Is this an answer to the question:

>  Is there any reason why shell_bt_nus use a logger instead of printk in main app ?

If so what is the reason?

Anyway keep it as LOG_INF for this PR and create a different PR for changing the nus sample to use printk instead if you wish to do that.

Reply:
We decided to use `printk` in the samples instead of the logger. The reason is that `printk` output is simple and consistent across projects which may have different configuration for the logger.
@KAGA164 can also pile a up a commit for that, instead of opening a new PR.

Reply:
I checked how this sample look with `printk`. I came to the conclusion that `LOG` look visually better than printk in this case. When `printk` is used then messages is printk with shell line mark.